pkg/limitador/k8s_objects.go

```diff
@@ -0,0 +1,141 @@
+package limitador
+
+import (
+	limitadorv1alpha1 "github.com/3scale/limitador-operator/api/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	DefaultVersion   = "latest"
+	ServiceName      = "limitador"
+	ServiceNamespace = "default"
```

Comment:
Will the limitador Service be always deployed to the `default` namespace, even when there are multiple limitadors deployed?
Depending on how Limitador/s are intended to be deployed this might need to change in the future

Reply:
Related to https://github.com/3scale-labs/limitador-operator/pull/2#discussion_r553881338 Not sure yet

Reply:
:+1: